CHANGELOG.md

```diff
@@ -41,12 +51,28 @@ request adding CHANGELOG notes for breaking (!) changes and possibly other secti
 - Added Catalog configuration for S3 and STS endpoints. This also allows using non-AWS S3 implementations.
 
 - The `IMPLICIT` authentication type enables users to create federated catalogs without explicitly
-providing authentication parameters to Polaris. When the authentication type is set to `IMPLICIT`, 
-the authentication parameters are picked from the environment or configuration files. 
+providing authentication parameters to Polaris. When the authentication type is set to `IMPLICIT`,
+the authentication parameters are picked from the environment or configuration files.
 
 - The `DEFAULT_LOCATION_OBJECT_STORAGE_PREFIX_ENABLED` feature was added to support placing tables
 at locations that better optimize for object storage.
 
+- Introduced S3 path-style access support for improved compatibility with S3-compatible storage systems.
+
+- Added MinIO integration support with comprehensive getting started documentation.
```

Comment:
Should we put this in the highlight section? I think this is a big thing for people using s3-compatible storage.

Reply:
I'm ok to move this to highlights, but s3-compatible support is not yet complete because of #2207... ATM, MinIO works, but systems without STS are not easy to use :shrug: